Formula/helmsman.rb

```diff
@@ -16,7 +17,6 @@ class Helmsman < Formula
 
   depends_on "go" => :build
   depends_on "helm"
-  depends_on "kubernetes-cli"
```

Comment:
no need to hard depend on k8s-cli

Reply:
but it's not calling that in the test so that means running the tool requires it.

Reply:
https://github.com/Praqma/helmsman#from-binary
> Please make sure the following are installed prior to using `helmsman` as a binary (the docker image contains all of them):
> 
> - [kubectl](https://github.com/kubernetes/kubectl)
> - [helm](https://github.com/helm/helm) (helm >=v2.10.0 for `helmsman` >= 1.6.0, helm >=v3.0.0 for `helmsman` >=v3.0.0)
> - [helm-diff](https://github.com/databus23/helm-diff) (`helmsman` >= 1.6.0)

Reply:
our build is not that different from the upstream.

helmsman is a utility tool to manage helm charts stuff (which depends on k8s cluster, that is the reason why it does need `kubernetes-cli`, but I think we can have kubernetes-cli installed separately, which is what we already did for helm formula)

Reply:
So if a user installs the tool, cam they use it without installing anything else? If the answer is no I'd say it's broken.

Reply:
it appears they do directly rely on the `kubectl` executable:
https://github.com/Praqma/helmsman/blob/a39171dcb5ef099ebb6b97c8a9b98d1c36dcd62a/internal/app/cli.go#L171-L173

helm is different because it uses the kubectl go api: https://github.com/helm/helm/blob/a9c957d35b6a2ce9a8fab03d7cb72414e99c240c/go.mod#L46

Reply:
> So if a user installs the tool, cam they use it without installing anything else? If the answer is no I'd say it's broken.

In that sense, yes.

OK, let me bring the k8s-cli runtime dep back.